words.html

```diff
@@ -915,29 +915,29 @@
 </p>
 <p id="conclusion_4">
 Natürlich wird Vertrauen in der echten Welt von viel mehr als diesen Faktoren beeinflusst.
-Es gibt Ansehen, gemeinsame Werte, Verträge, kulturelle Marker, blah blah blah.
-Und, das wir's nicht vergessen...
+Es gibt Ansehen, gemeinsame Werte, Verträge, kulturelle Marker und so weiter.
```

Comment:
Auch hier wieder: Ich will nah am Original bleiben. Das "blah blah blah" ist in beiden Sprachen sehr umgangsprachlich, aber es ist halt das, was der Autor geschrieben hat.